src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/Form.js

```diff
@@ -92,6 +93,13 @@ class Form extends React.Component<Props> {
                 : new ResourceStore(resourceKey, id, {locale}, formStoreOptions);
         } else {
             this.resourceStore = resourceStore;
+
+            if (Object.keys(this.resourceStore.data).length > 0) {
+                // data should be reloaded if ResourceTabs ResourceStore is used and user comes back from another tab
+                // the above check assumes that loading the data from the backend takes longer than calling this method
+                // the very unlikely worst case scenario if this assumption is not met, is that the data is loaded twice
+                this.resourceStore.load();
```

Comment:
@chirimoya I have changed this now as we discussed yesterday. It works, with the only drawback mentioned in the comments above. This assumption is even met on my machine, when running Sulu locally, so if there is even more network latency that should be no problem. I've put the comment here, so that we remember this if we should be experiencing double requests.

And this way of handling it also solves the problem you've mentioned yesterday, the data is reset when you come back to the tab in the same way as if you are switching to another tab.